src/main.rs

```diff
@@ -36,7 +36,7 @@ fn main() {
     let mut sample = mutate(&reference, vcf);
     let sample_end = SystemTime::now();
 
-    let target_gene = "mmpL5";
+    let target_gene = "Rv1096";
```

Comment:
why does this change?

Reply:
The main here is actually only used for debugging. The gene in the case I was looking at was `Rv1096` so this allowed for easy checking of outputs
I know I could set this up as a proper CLI application, but I'm not convinced that's needed